src/Core/src/MauiContext.cs

```diff
@@ -72,6 +72,20 @@ public void AddSpecific(Type type, Func<object, object?> getter, object state)
 			{
 				_scopeStatic[type] = (state, getter);
 			}
+
+			public object? GetKeyedService(Type serviceType, object? serviceKey)
+			{
+				if (Inner is IKeyedServiceProvider provider)
+					return provider.GetKeyedService(serviceType, serviceKey);
```

Comment:
No braces?

Reply:
We don't always for single line statements - especially in short methods.